scripts/solve_network.py

```diff
@@ -92,6 +92,7 @@
 pypsa.pf.logger.setLevel(logging.WARNING)
 
 from vresutils.benchmark import memory_logger
+from _helpers import mocksnakemake
```

Comment:
Is there a reason why you don't import it in the `if 'snakemake' not in globals():` block as elsewhere?

Reply:
no actually not, you are right one should keep it consistent having it always in the `if` block